### PR TITLE
phpExtensions: little cleanup

### PR DESCRIPTION
**Pull request by SuperSandro2000**
###### Description of changes

While searching for a good example to base my php-systemd PR on I found some things to cleanup.

###### Things done

- Built on platform(s)
  - [ ] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.11 Release Notes (or backporting 22.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2211-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Review by drupol — APPROVED** [review]

File: pkgs/development/php-packages/pinba/default.nix [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
-buildPecl {
+buildPecl rec {
   pname = "pinba";
   version = "1.1.2";
 
   src = fetchFromGitHub {
     owner = "tony2001";
     repo = "pinba_extension";
-    rev = "RELEASE_1_1_2";
+    rev = "RELEASE_${lib.replaceString ["."] ["_"] version}";
```
Comment (etu) [inline pkgs/development/php-packages/pinba/default.nix:10]:
```
error: attribute 'replaceString' missing

       at /var/lib/ofborg/checkout/repo/38dca4e3aa6bca43ea96d2fcc04e8229/mr-est/ofborg-evaluator-4/pkgs/development/php-packages/pinba/default.nix:10:22:

            9|     repo = "pinba_extension";
           10|     rev = "RELEASE_${lib.replaceString ["."] ["_"] version}";
             |                      ^
           11|     sha256 = "0wqcqq6sb51wiawa37hbd1h9dbvmyyndzdvz87xqji7lpr9vn8jy";
       Did you mean replaceStrings?
```

**Review by etu — CHANGES_REQUESTED** [review]
Still breaks eval :slightly_smiling_face:

File: pkgs/development/php-packages/pinba/default.nix (hunk 1 above)
Comment (drupol) [inline pkgs/development/php-packages/pinba/default.nix:10]:
```suggestion
    rev = "RELEASE_${lib.replaceStrings ["."] ["_"] version}";
```

**Comment by drupol** [comment]
@SuperSandro2000 It would be nice to have before branch off, do you want me to take over?

**Review by drupol — APPROVED** [review]

**Pull request merged**